.gitignore

```diff
@@ -31,6 +31,7 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.yarn/
```

Comment:
I updated my nodejs and got the latest yarn version which handles things a bit differently. That's for another PR sometime though